server/raft_transport.go

```diff
@@ -190,8 +190,10 @@ func (t *rpcTransport) processQueue(raftNodeID proto.RaftNodeID) {
 		}
 
 		if !client.IsHealthy() {
-			log.Warningf("raft client for node %d unhealthy", nodeID)
-			return
+			if log.V(1) {
```

Comment:
this will be spammy, the old code would return which would let the next message wait for the client to become ready (which has its own issues right now, see above)